scripts/generator.sh

```diff
@@ -33,127 +35,60 @@ elif [ "$1" = "server" ]; then
 
     cargo run --bin namadan -- --base-dir $NAMADA_BASE_DIR/setup/validator-0/.namada/ ledger
 elif [ "$1" = "client" ]; then
+    if test -f "$NAMADA_BASE_DIR_FILE"; then
+        NAMADA_BASE_DIR="$(cat $NAMADA_BASE_DIR_FILE)" 
```

Comment:
Thanks, my bad. This issue was caused by the inclusion of `namada = { path = "../shared", features = ["testing"] }` and `namada_apps = { path = "../apps", features = ["testing"] }` in the `[dependencies]` section of the `benches` crate. Essentially, this was causing the `testing` feature and hence the `dev` feature to be enabled when `namadan` was built. This ultimately meant that the genesis file specified in the `init-network` command was ignored.

I've fixed the script by adding `--package namada_apps --manifest-path Cargo.toml` to the `cargo run --bin namadan` command so that those features are no longer enabled. The client commands all seem to be working again.

Reply:
ah, I think that's my fault as I didn't realize that changing the benches deps would also affect `cargo run`. I wish we could use `dev-dependencies` for it :/ anyway, thank you fixing this!

Reply:
I've opened a PR to avoid building binaries with "testing"+"dev" features by default https://github.com/anoma/namada/pull/1955. We don't have to do anything here, but `--package namada_apps --manifest-path Cargo.toml` should no longer be required